gamestonk_terminal/portfolio/portfolio_model.py

```diff
@@ -421,7 +421,7 @@ def get_beta_text(df: pd.DataFrame) -> str:
     low = betas.idxmin(axis=1)
     string = (
         "Beta is how strongly a portfolio's movements correlate with the market's movements."
-        " A stock with a high beta is considered to be riskier, with the average being one."
+        " A stock with a high beta is considered to be riskier. The market has a beta of one."
```

Comment:
just get rid of "The market has a beta of one"

The statement above (Beta is how strongly correlated .... ) implies that correlation of 1 is the market